girder/api/v1/group.py

```diff
@@ -247,7 +247,7 @@ def inviteToGroup(self, group, userToInvite, level, quiet, force):
                 })
                 mail_utils.sendEmail(
                     to=userToInvite['email'], text=html,
-                    subject="Girder: You've been invited to a group")
+                    subject="%s: You've been invited to a group" % SettingKey.BRAND_NAME)
```

Comment:
`SettingKey.BRAND_NAME` is equal to `'core.brand_name'`. I think you need to use the setting model API to `get` the setting from the database here.